### PR TITLE
Draft for unified build script

### PR DESCRIPTION
**Pull request by SinTh0r4s**
Small todo to use unified build scripts of GT++ and Witchery++

### CODE REVIEW

File: repositories.gradle [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        name = "UsrvDE"
+        url = "http://jenkins.usrv.eu:8081/nexus/content/repositories/releases/"
+    }
+    maven { // Baubles
+        name = "UsrvDE_3rdParty"
+        url = "http://jenkins.usrv.eu:8081/nexus/content/repositories/thirdparty/"
+    }
+    maven {
+        name = "chickenbones"
+        url = "http://chickenbones.net/maven/"
```
Comment (glowredman) [inline repositories.gradle:14]: HTTPS

File: repositories.gradle [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    maven {
+        name = "gt"
+        url = "https://gregtech.overminddl1.com/"
+    }
+    maven { // TConstruct
+        name = 'DVS1 Maven FS'
+        url = 'http://dvs1.progwml6.com/files/maven'
```
Comment (glowredman) [inline repositories.gradle:22]: HTTPS

File: repositories.gradle [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    maven {
+        url = "http://maven.ic2.player.to/"
```
Comment (glowredman) [inline repositories.gradle:25]: HTTPS

File: repositories.gradle [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        name = "ic2"
+    }
+    maven {
+        url = "http://www.ryanliptak.com/maven/"
```
Comment (glowredman) [inline repositories.gradle:29]: HTTPS

File: repositories.gradle [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        name = "applecore"
+    }
+    maven {
+        url = "http://maven.cil.li/"
```
Comment (glowredman) [inline repositories.gradle:33]: HTTPS

**Comment by SinTh0r4s** [comment]
I removed the unused DepLoader from the core mods. Does anyone sees a problem with it?

**Comment by draknyte1** [comment]
> I removed the unused DepLoader from the core mods. Does anyone sees a problem with it?

Isn’t it used for downloading mods when a user grabs the pack from Curse?

**Comment by boubou19** [comment]
yeah deploader must stays until we drop curse support

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**